k8s/env.example.sh

```diff
@@ -6,7 +6,6 @@
 # VERSION can be a git ticket branch but with _ instead of /
 # example: tickets_DM-7139, or dev
 VERSION=1bef3d1
```

Comment:
Can we also set Git commit SHA to VERSION ?

Reply:
Yes it'll work, I'll fix the comment thanks :-)